pyhealth/unittests/test_datasets/test_eicu.py

```diff
@@ -0,0 +1,147 @@
+import datetime
+from sqlite3 import Timestamp
+import unittest
+from pyhealth.data.data import Event
+
+from pyhealth.datasets import eICUDataset
+from pyhealth.unittests.test_datasets.utils import EHRDatasetStatAssertion
+
+
+class eICUDataset(unittest.TestCase):
+
+    # to test the file this path needs to be updated
+    ROOT = "~/Code/PyHealth_sunlab/PyHealth/datasets/eicu-collaborative-research-database-demo-2.0.1"
```

Comment:
emmm, I will let patrick to help upload this demo version https://physionet.org/content/eicu-crd-demo/2.0.1/.

Reply:
Thanks @pat-jj. Please try this https://storage.googleapis.com/pyhealth/eicu-demo/

Reply:
https://physionet.org/content/eicu-crd-demo/2.0.1/